lib/ansible/modules/web_infrastructure/jira.py

```diff
@@ -243,7 +244,8 @@ def request(url, user, passwd, timeout, data=None, method=None):
     # resulting in unexpected results. To work around this we manually
     # inject the basic-auth header up-front to ensure that JIRA treats
     # the requests as authorized for this user.
-    auth = base64.encodestring('%s:%s' % (user, passwd)).replace('\n', '')
+    auth = to_text(base64.b64encode('{0}:{1}'.format(user, passwd).encode()))
```

Comment:
Use to_bytes instead of encode()
``` python
auth = to_text(base64.b64encode(to_bytes('{0}:{1}'.format(user, passwd), errors='surrogate_or_strict')))
```

Reply:
Why was replace('\n', '') removed?  If that was required before I don't see anything that would replace it now.

Reply:
`base64.encodestring(s) returns a string containing one or more lines of base64-encoded data always including an extra trailing newline ('\n')` which is not the case for b64encode and hence we do not need it anymore